tests/test_oml/test_datasets/test_list_dataest.py

```diff
@@ -0,0 +1,57 @@
+import csv
```

Comment:
as for tests in general, I believe we need one more test, when part of the images have boxes, and part of them don't